docs/services/slam/_index.md

```diff
@@ -14,114 +15,25 @@ Stability is not guaranteed.
-
-Add a service with type `slam`, whatever name you want, and the model of the library you want to use.
-
-Use [this JSON template](#integrated-library-orb-slam3) to configure the service with the ORB-SLAM3 library, and [this JSON template](#integrated-library-cartographer) to configure the service with the Cartographer library.
-Then, adjust general attributes and library-specific `config-params`.
+Integrated SLAM Libraries include:
 
-### General Attributes
+| Model | Description |
+| ----- | ----------- |
+| [`cartographer`](cartographer) | [The Cartographer Project](https://github.com/cartographer-project) performs dense SLAM using LIDAR data. |
+| [`orbslamv3`](orbslamv3) | [ORB-SLAM3](https://github.com/UZ-SLAMLab/ORB_SLAM3) performs sparse SLAM using monocular or RGB-D images. |
 
-| Name | Data Type | Inclusion | Description |
-| ---- | --------- | --------- | ----------- |
-| `data_dir` | string | **Required** | This is [the data directory](#data_dir-data-directory) used for saving input <file>sensor/map</file> data and output <file>map/</file> visualizations. Must be structured as specified [here](#data_dir-data-directory). |
-| `sensors` | string[] | **Required** | Names of configured [camera components](../../components/camera/) providing data to the SLAM service. |
-| `use_live_data` | bool | **Required** | <p>This specifies whether to run in Live or Offline mode.</p> <ul> `true`: Live mode. The service grabs the most recent sensor readings and uses those to perform SLAM. If no `sensors` are provided, an error will be thrown. </ul><ul>`false`: Offline mode. The service uses image data stored in the [data directory](#data_dir-data-directory) to perform SLAM.</ul> |
-| `map_rate_sec` | int | Optional | Map generation rate for saving current state *(seconds)*. <ul> Default: `60`. </ul> |
-| `data_rate_msec` | int | Optional | Data generation rate for collecting sensor data to feed to SLAM *(milliseconds)*. <ul>Default: `200`.</ul> |
-| `port` | string | Optional | Port for SLAM gRPC server. If running locally, this should be in the form "localhost:<PORT>". If no value is specified a random available port is assigned. |
-| `delete_processed_data` | bool | Optional | <p>Setting this to `true` helps to reduce the amount of memory required to run SLAM.</p> <ul> `true`: sensor data is deleted after the SLAM algorithm has processed it. </ul><ul> `false`: sensor data is not deleted after the SLAM algorithm has processed it. </ul> |
-| `config_params` |  map[string] string | Optional | Parameters specific to the `model` of SLAM library. |
-
-{{% alert title="Caution" color="caution" %}}
-
-- If `use_live_data: true`, `delete_processed_data: true` by default.
-- If `use_live_data: false`, `delete_processed_data: false` by default.
-
-Setting `delete_processed_data: true` and `use_live_data: false` is invalid and will result in an error.
-{{% /alert %}}
+## Data and Mapping Specifications
 
-### `config_params`
+### `data_dir`
```

Comment:
This is a config parameter, right? Should that just live in the configuration pages of each model? And for mapping modes also - these are here now completely without context

Reply:
Also line 38 - "to recap" is a strange way to start that sentence

Reply:
@npentrel resolved and removed!